src/ml_tooling/transformers.py

```diff
@@ -39,10 +39,23 @@ class FillNA(BaseEstimator, TransformerMixin):
     Fills NA values with given value or strategy. Either a value or a strategy has to be supplied.
     """
 
-    def __init__(self, value: int = None, strategy: str = None):
-        self.value = value
-        self.strategy = strategy
-        self.column_values_ = None
+    @classmethod
```

Comment:
Sorry, I was unclear - a class method is just a method on a class - @classmethod is a method that is given an uninstantiated class as the first argument.

```suggestion
    def validate_input(self, value, strategy)
```

Though we might have to test this, as the scikit-learn API contract might be violated by validating in __init__ instead of fit - have had some problems with this approach recently. After some research last night, I think sklearn requires you to not do any validation in the __init__, just straight assign, as it can't reconstruct the transformer when parallelizing.